source/features/default-to-rich-diff.tsx

```diff
@@ -42,6 +37,7 @@ function init(): void {
 
 features.add({
 	id: 'default-to-rich-diff',
+	description: 'Some of the files in pull requests can be reviewed using a rich-diff mode, for example, Markdown and SVG. This feature sets the rich-diff view as default in pull requests for files with a rich-diff.',
```

Comment:
I was able to merge the commit suggestion here, probably because I have access to the current repo.

I wasn't able to do this on forks recently as it errored out.

I already reported that bug to GitHub

Reply:
> Perhaps it can be dropped and we can replace the period in the middle with a semicolon

I don't really see it being more readable with a semicolon.

Reply:
> I don't really see it being more readable with a semicolon.

The suggestion was more about being able to drop the trailing period. If there are periods in the middle I think there should be one at the end too. If there are semicolons we might not need the trailing one

Reply:
Yes. I'm just not sure why it's so important to remove the trailing period?

Reply:
It's not. Except for consistency? 😅

Reply:
Doesn't matter. We might want to have multiple sentences for other features too. This feels like code-golfing descriptions.